packages/form-js-viewer/test/spec/util/GetSchemaVariables.spec.js

```diff
@@ -39,7 +40,7 @@ describe('util/getSchemaVariables', () => {
 
     const variables = getSchemaVariables(dynamicSchema);
 
-    expect(variables).to.eql([ 'product', 'xyzData', 'mailto', 'language', 'tags' ]);
+    expect(variables).to.eql([ 'xyzData', 'product', 'mailto', 'language', 'tags' ]);
```

Comment:
So getSchemaVariables used to linearly iterate through all the components, then search for the keys, and other variables on other properties. This however does not work now because something that's nested within a group isn't a top level key anymore. 

Like group(path: a) > checkbox(key: c), should not export c as a variable as it's a property of root object a. 

So now we iterate through the paths / keys as a separate step from the iteration through all the other properties, which changes the order in which they are aggregated to the array.